code/modules/reagents/chemistry/reagents/drugs.dm

```diff
@@ -670,29 +753,41 @@
 	addiction_chance = 10
 	addiction_threshold = 5
 	metabolization_rate = 0.6
+	addiction_decay_rate = 0.1 //very low to force them to take time off of meth
 	taste_description = "wiper fluid"
 
+/datum/reagent/lube/ultra/on_mob_add(mob/living/L)
+	ADD_TRAIT(L, TRAIT_GOTTAGOFAST, id)
+	if(ishuman(L))
+		var/mob/living/carbon/human/H = L
+		H.physiology.stun_mod *= 1.5
+
 /datum/reagent/lube/ultra/on_mob_life(mob/living/M)
 	var/update_flags = STATUS_UPDATE_NONE
+	var/recent_consumption = holder.addiction_threshold_accumulated[type]
+	M.AdjustJitter(10 SECONDS)
+	update_flags |= M.adjustStaminaLoss(-40, FALSE)
+	M.SetSleeping(0)
+	M.SetDrowsy(0)
+	if(prob(12 * DRAWBACK_CHANCE_MODIFIER(recent_consumption))) // slightly higher prob than meth due to the no nutrition thing
```

Comment:
Pound for pound, meth is better than lube, since 1) it has lower brain damage 2) the nutrition loss is really insignificant (there's plenty of food to go around) 3) organics can chug mannitol to lower the brain damage even further.

I don't think having more severe drawback here is well-justified.

Separately, i'm pretty sure IPCs use the exact same `nutrition` var / `adjust_nutrition` proc as organics, they just eat different stuff (i.e. APCs) to replenish it. You can add the increased hunger here, and it would work bothe mechanically and thematically (IPCs overclocking themselves and using more power)

Reply:
nutrition loss isn't insignificant mid combat or if you are an antagonist on the run, you dont exactly always have time to take a pitstop to eat food.